app/views/embedded/petitions/show.html.slim

```diff
@@ -0,0 +1,63 @@
+doctype html
+html
+  body.petition-widget
+    div.petition-container class="#{params[:size] || "big"}"
+      div.petition-header style="background-image: url('#{cycle.picture.url}')"
+        div.petition-header-overlay
+        div.petition-header-info
+          h3= phase.name
+          p= phase.description
+          = image_tag "watermark"
+      div.petition-progress-container
+        - if phase.finished?
+          div.petition-progress style="background-color: #{cycle.color};"
```

Comment:
Oops, I forgot to see how that would look like on the small version, it gets kinda weird because the button has the same color. I think using `rgba(0, 0, 0, 0.54)` instead will make it better